### PR TITLE
feat: add Dolphin and Konsole to the list

### PR DESCRIPTION
**Pull request by smirkybg**
Signed-off-by: Bogomil Vasilev <smirky@smirky.net>

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -137,6 +141,7 @@ <h2>Mostly...</h2>
             <a href="https://sw.kovidgoyal.net/kitty/">Kitty</a>,
             <a href="https://launchpad.net/sakura">Sakura</a>,
             <a href="https://github.com/thestinger/termite">Termite</a>
+            <a href="https://github.com/KDE/konsole">Konsole</a>
```
Comment (mpsq) [inline src/index.html:144]:
```suggestion
            <a href="https://sw.kovidgoyal.net/kitty/">Kitty</a>,
            <a href="https://github.com/KDE/konsole">Konsole</a>,
            <a href="https://launchpad.net/sakura">Sakura</a>,
            <a href="https://github.com/thestinger/termite">Termite</a>
```

**Review by mpsq — CHANGES_REQUESTED** [review]
Minor change: sorting the list alphabetically. It looks good otherwise, thanks for your contribution :muscle:.

**Review by mpsq — APPROVED** [review]

**Comment by smirkybg** [comment]
You seem to ditched my contribution to your list little after merging it. Something related to the webserver change I suppose?

**Comment by mpsq** [comment]
Thanks, I just fixed it :)

**Pull request merged**